app/src/main/java/com/example/trainingintive/repository/local/DogImageDao.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.trainingintive.repository.local
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.example.trainingintive.model.DogImageUrl
+
+@Dao
+interface DogImageDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insert(dogImageUrl: DogImageUrl)
+
+    @Query("SELECT * from dogImageUrl")
+    fun getAllDogImageUrls(): List<DogImageUrl>
```

Comment:
Minor: The name of this Dao is `DogImageDao` so names like this can be simplified to `getImageUrls` or `getAllImageUrls` because it is logical that this Dao is responsible for Dog Images